compiler/rustc_middle/src/ty/query/plumbing.rs

```diff
@@ -259,52 +256,54 @@ macro_rules! define_queries {
 
         #[allow(nonstandard_style)]
         #[derive(Clone, Debug)]
-        pub enum Query<$tcx> {
-            $($(#[$attr])* $name($($K)*)),*
+        pub struct Query {
+            pub name: &'static str,
+            hash: Fingerprint,
```

Comment:
Can we explain what this hash is of in a comment?

Reply:
Done.